src/ui/components/filters/sortoptionscomponent.js

```diff
@@ -47,40 +47,30 @@ export default class SortOptionsComponent extends Component {
 
   onMount () {
     // Handle radio button selections
-    DOM.on(
-      DOM.query(this._container, '.yxt-SortOptions-fieldSet'),
-      'change',
-      evt => this.handleOptionSelection(parseInt(evt.target.value))
+    const containerEl = DOM.query(this._container, '.yxt-SortOptions-fieldSet');
+    containerEl && DOM.on(containerEl, 'change', evt =>
+      this.handleOptionSelection(parseInt(evt.target.value))
```

Comment:
Every argument to DOM.on should be on its own line since not all the params can fit on one line

Reply:
updated!